cmd/crc/cmd/delete.go

```diff
@@ -5,12 +5,14 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"path/filepath"
 
 	"github.com/crc-org/crc/pkg/crc/constants"
 	crcErrors "github.com/crc-org/crc/pkg/crc/errors"
 	"github.com/crc-org/crc/pkg/crc/input"
 	"github.com/crc-org/crc/pkg/crc/logging"
 	"github.com/crc-org/crc/pkg/crc/machine"
+	crcOs "github.com/crc-org/crc/pkg/os"
```

Comment:
`crcOs` looks bit wired let's use  `crcos` which we are using in other packages also ie.
https://github.com/crc-org/crc/blob/da66c4e43c4dfac8878b5617aa43e7eaf955330f/pkg/crc/adminhelper/hosts_unix.go